Enable stddev for 2D surface to 1D spectrum.

e-koch opened the pull request:
Updates for MVC, SCF, VCA, PSpec. Enabling ```return_stddev``` gives errors on the 1D power spectrum.

Also added in saving for SCF, based off the pickling used for the dendrograms.

low-sky commented:
This looks good (in my cursory review).  However, I wanted to point out `scipy.stats.binned_statistic` as amazing for this kind of analysis in the unlikely scenario that you weren't aware of it.

keflavich commented:
Can the PSD and pspec1d stuff here now completely replace the agpy versions (https://github.com/keflavich/agpy/blob/master/AG_fft_tools/psds.py)?  If so, I'll add deprecation notices there and have them point to this repo.

e-koch commented:
@low-sky - Actually I didn't. Looks pretty cool, and we should keep it in mind if we need to do a re-write of the radial profiling.

@keflavich - Possibly. There's minimal changes between the two (I added logspaced bins and changed the stddev output a bit). Up to you, but I'd be fine if future changes are directed to here.
